airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -377,7 +377,7 @@ def read_records(self, stream_slice: Optional[Mapping[str, Any]] = None, **kwarg
 
     def request_params(self, stream_slice: Mapping[str, Any] = None, **kwargs):
         params = super().request_params(stream_slice=stream_slice, **kwargs)
-        params["expand[]"] = "data.discounts"
+        params["expand[]"] = ["data.discounts", "data.taxes"]
```

Comment:
doesn't this change the output schema? if so, we should update the `.json` schema file for this stream in order for normalization to process it correctly

Reply:
Schema was already in `.json`, you can check [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-stripe/source_stripe/schemas/checkout_sessions_line_items.json#L119), for some reason developer forgot to add this param. That's all.